packages/skeleton-svelte/src/lib/components/InputChips/InputChips.svelte

```diff
@@ -0,0 +1,66 @@
+<script lang="ts">
+	import * as tagsInput from '@zag-js/tags-input';
+	import { useMachine, normalizeProps } from '@zag-js/svelte';
+	import { useId } from '$lib/internal/use-id.js';
+	import type { TagsInputProps } from './types.js';
+
+	// Props
+	let {
+		value = $bindable(),
+		base = 'input',
+		inputBase = 'bg-transparent border-transparent focus:border-transparent focus:ring-0',
+		inputClasses = '',
+		chipContainerBase = 'px-3 py-2 flex gap-1',
+		chipContainerClasses = '',
+		chipBase = 'preset-filled-surface-950-50 px-2 py-1 rounded',
+		chipClasses = '',
+		editInputBase = 'p-0 bg-transparent leading-3 border-transparent focus:border-transparent focus:ring-0',
+		editInputClasses,
+		classes = '',
+		...zagProps
+	}: TagsInputProps = $props();
+
+	// Machine
+	const [snapshot, send] = useMachine(
+		tagsInput.machine({
+			id: useId(),
+			onValueChange: (details) => {
+				value = details.value;
+			}
+		}),
+		{
+			context: {
+				...zagProps,
+				get value() {
+					return $state.snapshot(value);
+				}
+			}
+		}
+	);
+
+	// API
+	const api = $derived(tagsInput.connect(snapshot, send, normalizeProps));
+</script>
+
+<div class="{base} {classes}" {...api.getRootProps()}>
```

Comment:
Don't forget to add comments denoting each part of the template. This helps match the style props up to each element. Container is a bit open ended so I'd suggest:

* Input
* List (instead of chipContainer)
* Chip
  * Label 
  * Button
  * Edit Input

Reply:
Done, again, feel free to edit.